go/stellar/loader.go

```diff
@@ -244,7 +244,21 @@ func (p *Loader) runRequests() {
 	}
 }
 
-func (p *Loader) loadPayment(id stellar1.PaymentID) {
+func (p *Loader) LoadPaymentSync(ctx context.Context, paymentID stellar1.PaymentID) {
+	mctx := libkb.NewMetaContext(ctx, p.G())
+	defer mctx.TraceTimed(fmt.Sprintf("LoadPaymentSync(%s)", paymentID), func() error { return nil })()
+
+	for i := 1; i <= 100; i++ {
+		err := p.loadPayment(paymentID)
+		if err == nil {
+			break
+		}
+		mctx.Debug("error on attempt %d to load payment %s: %s. sleep and retry.", i, paymentID, err)
+		time.Sleep(1 * time.Second)
```

Comment:
what about using libkb/backoff.go?

100 attempts sounds like a lot...

Reply:
done. in this case, the fastest i've seen it load is a few seconds, but it's typically closer to 7.